Sources/Ink/Internal/List.swift

```diff
@@ -93,12 +124,19 @@ internal struct List: Fragment {
     func html(usingURLs urls: NamedURLCollection,
               modifiers: ModifierCollection) -> String {
         let tagName = isOrdered ? "ol" : "ul"
+        
+        let startAttr: String
+        if let startingIndex = startingIndex, startingIndex != 1 {
+            startAttr = #" start="\#(startingIndex)""#
```

Comment:
Does this really belong in the HTML?

Reply:
This is how you indicate the first items index to the web browser, yes.

Reply:
Ah, cool. I didn't know that 👍